### ISSUE TITLE
Features request: More focus to productivity

### ISSUE THREAD

**Issue by AhmadMS1988** [linuxmint/cinnamon#9585]
Dears in linux mint, thank you for your efforts in maintaining such a wonderful linux distribution and desktop environment. I am a linux mint user since Linux mint 3 Cassandra, and your work is really magnificent.
I want want to drew your attention to some tweaks that may improve the usability of the OS:
1- Please extend the calendar applet functionality to display the events in a selected day. or you can introduce a notifications or action panel for the desktop.
2- Please do not focus more thunderbird, and try introduce a new mail client, or integrate geary more into your system or even introduce a new mail client. Thunder bird UI is pretty old, also all custom integration needs plugins and very hard to maintain especially for exchange and google gsuite. Geary and calendar on the other hand are well integrated with Gnome online accounts, and your touch may improve it more.
3- Better workspaces management out of the box without the need to have extensions installed to control that.
4- GSConnect replacement or integration.
5- I think that IntelliJ products are used widely in. the software community, and IntelliJ toolbox is a very good way to control these tools. since Linux mint 19, I always report to IntelliJ a bug that the toolbox does not always appear in the tray and they did not fix it until now, it might be helpful if you can investigate if you can help from your side to fix this issue.
6- please backport the fingerprint functionality of gnome 3.38 ASAP.

Most of these features are available in Fedora 32, but what hold me there was the lack of support of the official docker community engine for 32 and it is less organizes and harder in package management.

To honest, since linux mint 3, I can't fully relay on linux distribution for my daily driver, and in the introduction of WSL 2 on windows, brings most of the missing advantages of linux to widows in addition to its superiority in usability, and small touches like these may improve the usability of linux mint.

Hope that some of may suggestion may be taken into consideration. and thank you again for your efforts.

**Comment by mfreeman72** [comment]
Agreed. Especially with the calendar applet, which I've asked about for way too many years.

**Comment by JoeJoeTV** [comment]
@AhmadMS1988 

To your fourth point: do you want a replacement for KDE Connect, or just a better interface on cinnamon?
If you are fine with installing KDE Connect, you could use [KDE Connect Control Center by Severga](https://cinnamon-spices.linuxmint.com/applets/view/330) or [KDE Connect Applet by me](https://cinnamon-spices.linuxmint.com/applets/view/334), which make most of the features available as an Applet for Cinnamon. 

[I also made a Desklet](https://cinnamon-spices.linuxmint.com/desklets/view/57), which shows notifications and some other info(more in the future)about a device, but it's not fully finished.
